### ISSUE TITLE
Multiple feature extractors in a single Cut

### ISSUE THREAD

**Issue by njellinas** [lhotse-speech/lhotse#1350]
Is it possible to have multiple feature extractors for a single Cut?
e.g. if you want mel spectrograms, F0, energy, etc. as separate features in a model, can you implement each one separately and call compute_features and get them as separate arrays/tensors?

**Comment by pzelasko** [comment]
Yes, one example is given towards the end of this tutorial https://github.com/lhotse-speech/lhotse/blob/master/examples/04-lhotse-shar.ipynb

if you don’t want to go into tar archives you can use any of the feature writers in a pattern like this: cut.my_feature = write.store_array(…)
